integration/fuse/src/main/java/alluxio/fuse/AlluxioJniFuseFileSystem.java

```diff
@@ -657,6 +664,26 @@ public String getFileSystemName() {
     return mFsName;
   }
 
+  @Override
+  public void umount() {
+    // Release operation is async, we need to make sure
+    // all out stream is closed before umount the fuse
+    LOG.info("Waiting for all file out stream closed");
```

Comment:
Can we wrap the try{} block if a mCreateFileEntries.isEmpty() check? so that we can log the info when we actually need to wait?

Reply:
fixed, thx